core/templates/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -69,6 +69,8 @@ export class StateEditorService {
   stateHintsEditorInitialised: boolean = false;
   stateSolutionEditorInitialised: boolean = false;
   stateEditorDirectiveInitialised: boolean = false;
+  _stateEditorInitializedEventEmitter = new EventEmitter();
```

Comment:
I meant something like this:
```suggestion
  private _stateEditorInitializedEventEmitter = new EventEmitter();
```